go/glome/glome.go

```diff
@@ -16,11 +16,10 @@
 package glome
```

Comment:
Nit: imho small changes in code that does not alter the functionality of the code should be avoided so that every commit is self-contained.

Reply:
SGTM too.